### PR TITLE
[Merged by Bors] - chore: empty ports of the Data/Buffer/Parser files

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Buffer/Parser/Numeral.lean [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Porting note:
+As the parsing framework has completely changed in Lean 4
+there is no point porting these files directly.
+They can be rewritten from scratch as needed, just as for tactics.
+-/
+
+-- Porting note:
+-- As the parsing framework has completely changed in Lean 4
+-- there is no point porting these files directly.
+-- They can be rewritten from scratch as needed, just as for tactics.
```
Comment (jcommelin) [inline Mathlib/Data/Buffer/Parser/Numeral.lean:20]:
```suggestion
```

**Comment by jcommelin** [comment]
The three porting headers do not all refer to the same commit hash. Is that on purpose?

File: Mathlib/Data/Buffer/Parser/Numeral.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/Data/Buffer/Parser/Numeral.lean:20]: Is the suggestion here to just drop the comment?

File: Mathlib/Data/Buffer/Parser/Basic.lean [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/-
+Copyright (c) 2020 Yakov Pechersky. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yakov Pechersky
+
+! This file was ported from Lean 3 source module data.buffer.parser.basic
+! leanprover-community/mathlib commit bb9d1c5085e0b7ea619806a68c5021927cecb2a6
```
Comment (semorrison) [inline Mathlib/Data/Buffer/Parser/Basic.lean:7]:
```suggestion
! leanprover-community/mathlib commit 2220b0cbab795e73674b8191170b0cc68c6b54a8
```

**Comment by semorrison** [comment]
> The three porting headers do not all refer to the same commit hash. Is that on purpose?

Not sure how that happened. mathlib3 updated underneath me? Anyway, they are all the same now.

File: Mathlib/Data/Buffer/Parser/Numeral.lean (hunk 1 above)
Comment (jcommelin) [reply Mathlib/Data/Buffer/Parser/Numeral.lean]: It's duplicated

**Comment by jcommelin** [comment]
bors d+

**Comment by fpvandoorn** [comment]
bors merge

**Pull request closed without merging**